source/localizable/_code-of-conduct.en.md

```diff
@@ -1,39 +1,39 @@
 Purpose
 -------
 
-A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+A primary goal of all the conferences and user groups that follow this Code of Conduct is to create an inclusive community that welcomes and supports as many people with various backgrounds and identities as possible. As such, we are committed to build a friendly and safer environment for all. These various backgrounds and identities include (but are not limited to) gender identity and expression, sex, sexual orientation, educational level, mental and physical ability, immigration status, race, ethnicity, national origin, social and economic status, age, size and religion (or lack thereof).
```

Comment:
@PragTob 

The previous text about 'regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof)' is as a standard text of cause the easiest way (and used in a lot of other coc, events, etc.), but if the Ruby/tech community wants to become more progressive/inclusive, in my opinion (as I understood also @lauralindal 's) it is important that we need to make extra efforts in the direction of reducing the barriers and power structures for underrepresented groups by addressing them explicitly with a more welcoming text (than just 2 lines), because their issues, backgrounds and identities are much more various than we know or can imagine.

Laura and me are not 100%ly sure about the wordings in the listing. This is why especially for this part it is highly recommended to ask organizations who have expertise in these fields.